src/utils/fileLoaders.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const path = require('path');
+const jsyaml = require('js-yaml');
+
+const cwd = process.cwd();
+
+const file = (refValue, options) => {
+    let refPath = refValue;
+    const baseFolder = options.baseFolder ? path.resolve(cwd, options.baseFolder) : cwd;
+
+    if (refPath.indexOf('file:') === 0) {
+        refPath = refPath.substring(5);
+    } else {
+        refPath = path.resolve(baseFolder, refPath);
+    }
+
+    const filePath = getRefFilePath(refPath);
+    const filePathLowerCase = filePath.toLowerCase();
+
+    var data = fs.readFileSync(filePath, 'utf8');
+    if (filePathLowerCase.endsWith('.json')) {
+        return JSON.parse(data);
+    } else if (filePathLowerCase.endsWith('.yml') || filePathLowerCase.endsWith('.yaml')) {
+        return jsyaml.load(data);
+    }
+};
+
+function getRefFilePath(refPath) {
+    let filePath = refPath;
```

Comment:
this assignment seems redundant, probably you can simply do early return instead

Reply:
changed